dist/css/main.css

```diff
@@ -979,4 +983,152 @@ footer ul li a img {
             transform: scale(1.5);
   }
 }
+
+.popup {
```

Comment:
- [ ] The popup needs to follow the design for both desktop and mobile versions from the chosen template in Figma (e.g., font, colors, images, margins), kindly change the close button ```color```, ```font-weight``` and add some ```padding``` to the buttons so your design be as close as possible to the Figma design

- Your design:
![Screenshot_20220511_222312](https://user-images.githubusercontent.com/25227022/167950060-af0bd7f1-e65b-4e14-87f2-889abe042ad6.png)

- Figma design:
![Screenshot_20220511_222329](https://user-images.githubusercontent.com/25227022/167950109-1ac074dc-73f4-4c55-86f6-5707b3bbef67.png)